ui_framework/src/components/flex/flex_grid.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const gutterSizeToClassNameMap = {
+  none: '',
+  small: 'kuiFlexGrid--gutterSmall',
+  medium: 'kuiFlexGrid--gutterMedium',
+  large: 'kuiFlexGrid--gutterLarge',
+  extraLarge: 'kuiFlexGrid--gutterExtraLarge',
```

Comment:
I'd be OK with `xLarge` here if you are. If we have to go up to XXL, typing "extraExtraLarge" could be kinda silly.